chainer/functions/connection/n_step_gru.py

```diff
@@ -93,9 +93,9 @@ def n_step_gru(
             hidden units.
         xs (list of chainer.Variable): A list of :class:`~chainer.Variable`
             holding input values. Each element ``xs[t]`` holds input value
-            for time ``t``. Its shape is ``(B_t, I)``, where ``B_t`` is
-            mini-batch size for time ``t``, and ``I`` is size of input units.
-            Note that this functions supports variable length sequences.
+            for time ``t``. Its shape is ``(L_t, I)``, where ``L_t`` is the
```

Comment:
I think the function node class gets transposed sequences. So, the original description is correct.

Reply:
Oh, I see. Thank you for your comment!